### ISSUE TITLE
Schema change job could not finished

### ISSUE THREAD

**Issue by kangkaisen** [apache/incubator-doris#1393]
**Describe the bug**
Just now， our user found a schema change job run a long time:
![image](https://user-images.githubusercontent.com/9894906/60170981-3075d880-983c-11e9-81d7-fc81b2ed5e31.png)

I found:

1 The clear alter task run successfully in BE

2 FE continuous send clear schema change job to BE:
```
2019-06-26 16:40:28,809 INFO 29 [SchemaChangeHandler.runOneCycle():1145] schema change job has finished, send clear tasks to all b
e SchemaChangeJob [tableName=app_dt__poi_cis_matched, type=schema change, state=FINISHING, dbId=10004, tableId=111004980, transact
ionId=312527717, isPreviousLoadFinished=true, createTime=1561532629855, finishedTime=-1]
2019-06-26 16:40:28,830 INFO 29 [SchemaChangeJob.checkOrResendClearTasks():332] sending clear schema change job tasks for table [1
11004980]
2019-06-26 16:40:28,841 INFO 29 [SchemaChangeJob.checkOrResendClearTasks():364] successfully sending clear schema change job [1110
04980]
```

3 FE cannot find some clear alter task:
```
2019-06-26 16:49:10,045 WARN 1620 [MasterImpl.finishTask():122] cannot find task. type: CLEAR_ALTER_TASK, backendId: 18710, signat
ure: 111071104
2019-06-26 16:49:10,046 WARN 293815 [MasterImpl.finishTask():122] cannot find task. type: CLEAR_ALTER_TASK, backendId: 63256540, s
ignature: 111054732
2019-06-26 16:49:10,047 WARN 2357 [MasterImpl.finishTask():122] cannot find task. type: CLEAR_ALTER_TASK, backendId: 18710, signat
ure: 111067009
2019-06-26 16:49:10,048 WARN 1951 [MasterImpl.finishTask():122] cannot find task. type: CLEAR_ALTER_TASK, backendId: 63256540, sig
nature: 111052684
2019-06-26 16:49:10,049 WARN 3968 [MasterImpl.finishTask():122] cannot find task. type: CLEAR_ALTER_TASK, backendId: 18710, signat
ure: 111058817
```

**Comment by morningman** [comment]
will be resolved in #1429

**Comment by kangkaisen** [comment]
This PR https://github.com/apache/incubator-doris/pull/1695 should fix this issue.
